src/menus/icon-menu.jsx

```diff
@@ -158,6 +158,7 @@ const IconMenu = React.createClass({
       },
       touchTapCloseDelay: 200,
       useLayerForClickAway: false,
+      iconStyle: {},
```

Comment:
Thanks @pdf! If we move forward with this, could you put this in the object in alphabetical order?